sympy/core/compatibility.py

```diff
@@ -38,12 +38,12 @@
 
 Moved modules:
     * `reduce()`
-    * `StringIO()`
-    * `cStringIO()` (same as `StingIO()` in Python 3)
+    * `cStringIO()` (uses `StringIO.cStringIO` in Python 2, same as `StingIO()`
+      in Python 3, use `io.StringIO` for normal `StringIO`)
     * Python 2 `__builtins__`, access with Python 3 name, `builtins`
 
 Iterator/list changes:
-    * `xrange` removed in Python 3, import `xrange` for Python 2/3 compatible
```

Comment:
I just did a grep, and it's not used anywhere. We used to use Python 2 idioms like `xrange`, but we changed to use Python 3 idioms like `range`.